R/pareto_smooth.R

```diff
@@ -0,0 +1,534 @@
+#' Pareto khat diagnostic
+#'
+#' Estimate Pareto k value by fitting a Generalized Pareto
+#' Distribution to one or two tails of x. This can be used to etimate
+#' the number of fractional moments. For further details see Vehtari
```

Comment:
the number of fractional moments that is useful for convergence diagnostics. For ...

Reply:
Added this